website/content/docs/v0.3/Guides/rpi4-as-servers.md

```diff
@@ -0,0 +1,253 @@
+
+## Change CPU to Max (optinally)
+
+1. From the home page, visit "Device Manager".
+2. Go down to "Rasphberry Pi Configuration" and open that menu.
+3. Go to "CPU COnfiguration".
+4. Change CPU clock to `Max`.
+
+## Change boot order
+
+1. From the home page, visit "Boot Maitenece Manager".
+2. Go to "Boot Options".
+3. Go to "Change Boot Order".
+4. Make sure that `UEFI PXEv4` is the first boot option.
+
+## Persisting changes
+
+Now that we have made the changes above, we need to persist these changes. Go
+back to the home screem and hit `reset` to save the changes to disk.
+
+When you hit `reset`, the settings will be saved to the `RPI_EFI.fd` file on the
+SD card. This is where we will run into a limitation that is explained in the
+following issue: https://github.com/pftf/RPi4/issues/59. What this mean is that
+we need to create a `RPI_EFI.fd` file for each Pi that we want to use as server.
+This is because that the MAC address is also stored in the `RPI_EFI.fd` file,
+which makes it invalid when you try to use it in a different Pi.
+
+Plug the SD card back into your computer and extract the `RPI_EFI.fd` file from
+it and place it into the `raspberrypi4-uefi/serials/<serial>/`. The dir should
+look like this:
+
+```bash
+raspberrypi4-uefi/
+├── pkg.yaml
+└── serials
+    └─── XXXXXXXX
+        └── RPI_EFI.fd
+```
+
+# Build the image with the boot folder contents
+Now that we have the `RPI_EFI.fd` of our Pi in the correct location, we must now
+build a docker image containing the boot folder for the EEPROM. To do this, run
+the following command in the pkgs repo:
+
+`make PLATFORM=linux/arm64 USERNAME=$USERNAME PUSH=true TARGETS=raspberrypi4-uefi`
+
+This will build and push the following image:
+`ghcr.io/$USERNAME/raspberrypi4-uefi:<tag>`
+
+_If you need to change some other settings like registry etc, have a look in the
+Makefile to see the available variables that you can override._
+
+The content of the `/tftp` folder in the image will be the following:
+
+```bash
+XXXXXXXX
+├── RPI_EFI.fd
+├── Readme.md
+├── bcm2711-rpi-4-b.dtb
+├── bcm2711-rpi-400.dtb
+├── bcm2711-rpi-cm4.dtb
+├── config.txt
+├── firmware
+│   ├── LICENCE.txt
+│   ├── Readme.txt
+│   ├── brcmfmac43455-sdio.bin
+│   ├── brcmfmac43455-sdio.clm_blob
+│   └── brcmfmac43455-sdio.txt
+├── fixup4.dat
+├── overlays
+│   └── miniuart-bt.dtbo
+└── start4.elf
+```
+
+# Patch metal controller
+To enable the 2 boot process, we need to include this EEPROM boot folder into
+the sidero's tftp folder. To achieve this, we will use an init container using
+the image we created above to copy the contents of it into the tftp folder.
+
+Create a file `patch.yaml` with the following contents:
+
+```yaml
+spec:
+  template:
+    spec:
+      volumes:
+        - name: tftp-folder
+          emptyDir: {}
+      initContainers:
+      - image: ghcr.io/<USER>/raspberrypi4-uefi:v<TAG> # <-- change accordingly.
+        imagePullPolicy: Always
```

Comment:
```suggestion
        imagePullPolicy: Always # this is here in case you re run the make command as the same image tag will be reused.. Once you are confident about a successful build, you can remove this.
```